Example/ViewController.swift

```diff
@@ -109,9 +111,11 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        // Trigger OfflineServiceManager singleton creation to make sure that Maps SDK is notified whenever offline packs are available.
+        let _ = OfflineServiceManager.instance
```

Comment:
OfflineServiceManager shared instance creation allows to notify Maps SDK/MBGLCore whether there are any offline packs which are already downloaded. Current OfflineService implementation is not able detect whether offline packs are available implicitly so without this check offline packs will not be visible.

I'm not really fond of using singleton, but it allows to prevent any errors with handling OfflineService lifecycle. I think another way of using `OfflineService` might involve its local instances which are created whenever they're needed (e.g. to list available packs etc), but major thing is that user has to *"login"* to `OfflineService` before `MGLMapView` object is created to be able to show offline packs correctly. 

We might consider this approach in wrapper implementation for OfflineService, which is tracked in another task. In any case let me know what you think about current implementation.

/cc @1ec5, @IodaMikeMapbox